charts/oes/values.yaml

```diff
@@ -719,37 +718,37 @@ spinnaker:
         healthEndpoint: /health
         kubernetes:
           useExecHealthCheck: false
-        artifactId: quay.io/opsmxpublic/ubi8-spin-gate:1.18.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-gate:1.20.0
       deck.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-deck:3.3.0
+        artifactId: quay.io/opsmxpublic/ubi8-spin-deck:3.5.1
       clouddriver.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:7.2.2
       clouddriver-caching.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:7.2.2
       clouddriver-rw.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:7.2.2
       clouddriver-ro.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:7.2.2
       clouddriver-ro-deck.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:7.2.2
       echo.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:2.14.0
+        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:2.15.2
       echo-scheduler.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:2.14.0
+        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:2.15.2
       echo-worker.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:2.14.0
+        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:2.15.2
       fiat.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-fiat:1.13.0
+        artifactId: quay.io/opsmxpublic/ubi8-spin-fiat:1.14.1
       front50.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-oes-front50:0.25.1-opa
+        artifactId: quay.io/opsmxpublic/ubi8-spin-front50:0.26.1-opa
```

Comment:
Question: Why did we use the oes front50 earlier?  Was it or the OPA policy while saving pipelines and now it is part of the regular spin image?

Reply:
My understanding is if we are building the images with the customized changes from our forked repo, they should be pushed to the repo ubi8-oes-<service>:<tag>, and if images are built from open source code without any customizations using ubi8 base image, they should be pushed to the repo ubi8-spin-<service>:<tag>
Correct me if my understanding is wrong.

In this case, I think Yugandhar pushed it to ubi8-spin-front50 instead of pushing it to ubi8-oes-front50.

Reply:
Addressed this, image is now ubi8-oes-front50:0.26.1